.github/workflows/pipeline.yml

```diff
@@ -148,3 +148,30 @@ jobs:
           project_id: 'logflare-staging'
       - name: 'Trigger Cloud Build'
         run: 'gcloud beta builds triggers run logflare-app-staging-trigger --branch=staging --format "value(name)"'
+  trigger_cloud_build_production:
+    name: Trigger Cloud Build in Production
+    if: github.ref == 'refs/heads/staging'
+    needs:
+      - publish_latest
+      - publish_version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - id: 'auth'
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: '${{ secrets.GCP_PROD_CREDENTIALS }}'
+          create_credentials_file: true
+          export_environment_variables: true
+          cleanup_credentials: false
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+        with:
+          version: '413.0.0'
+          install_components: 'beta'
+          project_id: 'logflare-232118'
+      - name: 'Trigger Cloud Build'
+        run: 'gcloud beta builds triggers run logflare-master --branch=staging --format "value(name)"'
```

Comment:
should be `master` branch

Reply:
also, the cloudbuild trigger names between prod and staging don't seem to be consistent.

on another note, we can also remove the old base build triggers which are no longer used anymore.

Reply:
will do that cleanup afterwards (aka after first deploy to prod)